src/main/java/pl/javastart/task/Room.java

```diff
@@ -0,0 +1,21 @@
+package pl.javastart.task;
+
+public class Room {
+    double measure;
+    double temperature;
+    double minTemperature;
+    boolean acMounted;
+
+    public Room(double measure, double temperature, double minTemperature, boolean acMounted) {
+        this.measure = measure;
+        this.temperature = temperature;
+        this.minTemperature = minTemperature;
+        this.acMounted = acMounted;
+    }
+
+    void coolDown() {
+        if (acMounted && (temperature > minTemperature)) {
+            temperature--;
+        }
+    }
```

Comment:
1. Metoda miała zwracać informację czy udało się obniżyć temperaturę:

> . Metoda powinna zwracać true, jeżeli udało się obniżyć temperaturę, lub false jeżeli w pokoju nie ma klimatyzatora lub osiągnięto temperaturę graniczną.

2. Jeśli temperatura to 20,5*C, a minimalna to 20*C, to po uruchomieniu metody będzie 19,5*C, czyli mniej niż minimalna.